website/pages/en/network/developing.mdx

```diff
@@ -2,52 +2,122 @@
+
+Learn how to [deploy subgraphs](/deploying-a-subgraph-to-studio/).
+
+## Publish to the Network
+
+When you are happy with your subgraph, you can publish it to The Graph Network.
+
+- This is an on-chain action, which registers the subgraph and makes it discoverable by Indexers.
+- Published subgraphs have a corresponding NFT, which is then easily transferable.
+- Published subgraphs have associated metadata, which provides other network participants with useful context and information.
+
+Learn how to [publish a subgraph](/publishing/publishing-a-subgraph/).
+
+## Add Signal for Indexing
+
+Published subgraphs are unlikely to be picked up by Indexers without signal. To encourage indexing you should add signal to your subgraph.
+
+**What is signal?**
+
+- Signal is locked GRT associated with a given subgraph. It indicates to indexers that a given subgraph will receive query volume and it contributes to the indexing rewards available for processing it.
+- Third party Curators may also signal on a given subgraph, if they deem the subgraph likely to drive query volume.
+
+Learn more about signaling and [curating](/network/curating/) on The Graph.
+
+## Querying & Application Development
+
+Once a subgraph has been processed by Indexers and is available for querying, you can start using your subgraph in your applications.
+
+- You query subgraphs via a gateway, which forwards your queries to an Indexer who has processed the subgraph (paying query fees in GRT).
+- In order to make queries, you must generate an API key, which can be done in [Subgraph Studio](/https://thegraph.com/studio/).
+- This API key must be funded with GRT to pay query fees.
+- You can set a maximum query fee to control their costs and limit your API key to a given subgraph or origin domain.
+- Subgraph Studio provides you with data on your API key usage over time.
+- You can express an Indexer preference to the gateway, such as preferring Indexers with fast query response or have up-to-date data. These controls are set in Subgraph Studio.
+
+Learn more about [querying](/querying/querying-the-graph/).
+
+## Updating Subgraphs
+
+To update your subgraph with bug fixes or new functionalities, initiate a transaction to point it to the new version. You can deploy new versions of your subgraphs to [Subgraph Studio](/https://thegraph.com/studio/) for development and testing.
+
+- If you selected "auto-migrate" when you applied the signal, updating the subgraph will migrate any signal to the new version and incur a migration tax.
+- This signal migration should prompt Indexers to start indexing the new version of the subgraph, so it should soon become available for querying.
+
+## Deprecating Subgraphs
+
+If you no longer need a published subgraph, then you will need to deprecate the subgraph.
+
+- Deprecating a subgraph returns any signaled GRT to Curators.
+- To deprecate a subgraph, see the [deprecating a subgraph guide](/network/developing/deprecating-a-subgraph/).
```

Comment:
I'm not sure about the use of a list here. This makes more sense to me:
```suggestion
If you no longer need a published subgraph, then you will need to deprecate the subgraph. Deprecating a subgraph returns any signaled GRT to Curators.

To deprecate a subgraph, see [Deprecating a subgraph](/managing/transfer-and-deprecate-a-subgraph/#deprecating-a-subgraph).
```
(I also fixed the URL, I used the new one from #741 – is that OK?)